src/vcpkg/base/json.cpp

```diff
@@ -256,7 +258,7 @@ namespace vcpkg::Json
     }
     Value Value::number(double d) noexcept
     {
-        vcpkg::Checks::check_exit(VCPKG_LINE_INFO, isfinite(d));
+        vcpkg::Checks::check_exit(VCPKG_LINE_INFO, std::isfinite(d));
```

Comment:
With math.h this change is no longer necessary.

I think metrics.cpp needs a math.h too

Reply:
Thanks for the merge with the branch, @BillyONeal

I checked metrics.cpp, and it's already including `<math.h>` in line 16.
https://github.com/microsoft/vcpkg-tool/blob/2e27bc753b5c9c5d7eda309dff9d0d7e9cbc99ba/src/vcpkg/metrics.cpp#L16

Reply:
Excellent, thanks. Can you revert the std:: here or would you like me to do it for you? (Strictly speaking, `math.h` only makes `foo` exist, not `std::foo`)

Reply:
```suggestion
        vcpkg::Checks::check_exit(VCPKG_LINE_INFO, isfinite(d));
```